staging/provisioning/windows/windowsnodereset.ps1

```diff
@@ -49,6 +49,17 @@ if ($global:EnableHostsConfigAgent) {
 # Perform cleanup
 #
 
+Write-Log "Cleaning up persisted HNS policy lists"
+# Workaround for https://github.com/kubernetes/kubernetes/pull/68923 in < 1.14,
```

Comment:
Should we remove these two workaround comments or say that the October patch impact ALL versions to make sure this doesn't get accidently cleaned up in the future?